packages/flutter/lib/src/rendering/error.dart

```diff
@@ -50,6 +51,9 @@ class RenderErrorBox extends RenderBox {
 
   /// The message to attempt to display at paint time.
   final String message;
+  final double systemTopPadding = Zone.current['systemTopPadding'] == null
```

Comment:
> You lose all of that with zones.

On the getter side, you can keep the `ui.window` getter with the appropriate type (as in the code snippet above) keep all that good stuff.  The setting side is tricker.

> Anyway I don't want to beat a dead horse.

Hopefully we haven't scared @xster :)